sasjs-tests/src/Login.tsx

```diff
@@ -11,7 +11,7 @@ const Login = (): ReactElement<{}> => {
   const handleSubmit = useCallback(
     (e) => {
       e.preventDefault()
-      appContext.adapter.logIn(username, password).then((res) => {
+      appContext.adapter.logIn(username, password, 'clientID1').then((res) => {
```

Comment:
is this still required?  We are deprecating clientId in web mode

Reply:
https://github.com/sasjs/adapter/pull/708